### PR TITLE
Add manual request example

### PR DESCRIPTION
**Pull request by xtuchyna**

### CODE REVIEW

**Review by fridex — APPROVED** [review]
:1st_place_medal:

File: kebechet/managers/update/README.rst [hunk 1]
```diff
@@ -72,7 +72,15 @@ An example configuration:
...
-You can see this manager in action `here <https://github.com/thoth-station/kebechet/pull/46>`_, `here <https://github.com/thoth-station/kebechet/pull/85>`_ or `here <https://github.com/thoth-station/solver/issues/38>`_.
+You can see this manager in action in these various scenarios:
+
+- `Automatic dependency update <https://github.com/thoth-station/kebechet/pull/46>`_
+
+- `Initial dependency lock <https://github.com/thoth-station/kebechet/pull/85>`_
+
+- `Dependency update failure report <https://github.com/thoth-station/solver/issues/38>`_
+
+- `Manual dependeny update request <https://github.com/thoth-station/mi/issues/227>`_
```
Comment (saisankargochhayat) [inline kebechet/managers/update/README.rst:83]: r(dependeny/dependency)

**Comment by saisankargochhayat** [comment]
One minor suggestion. Thanks :D

**Review by saisankargochhayat — APPROVED** [review]
Thanks :100:

**Pull request merged**